pkg/services/ngalert/models/alert_rule.go

```diff
@@ -58,8 +58,10 @@ type AlertRule struct {
 	Updated         time.Time
 	IntervalSeconds int64
 	Version         int64
-	UID             string `xorm:"uid"`
-	NamespaceUID    string `xorm:"namespace_uid"`
+	UID             string  `xorm:"uid"`
+	NamespaceUID    string  `xorm:"namespace_uid"`
+	DashboardUID    *string `xorm:"dashboard_uid"`
+	PanelID         *int64  `xorm:"panel_id"`
```

Comment:
Is there a particular reason why you use pointers here and do not use them in other structs?

Reply:
It's because the columns in the table are nullable so we need to be able to represent `nil` in the model.

Reply:
It's because the columns in the table are nullable so we need to be able to represent `nil` in the model.